src/components/Navbar/components/CurrentLocationWidget/CurrentLocationWidget.tsx

```diff
@@ -0,0 +1,55 @@
+import { Paths } from 'enums/Paths';
+import { useLocation } from 'react-router-dom';
+import { CurrentLocationWidgetDesktop } from './components/CurrentLocationWidgetDesktop';
+import { CurrentLocationWidgetMobile } from './components/CurrentLocationWidgetMobile';
+import { Container, WeatherIcon, Description, DegreeSign } from './CurrentLocationWidget.styled';
+
+const CurrentLocationWidget = () => {
+  const location = useLocation();
+  let isLoading = true;
+  const data: any = {}; // temporary solution in order not to use
+
+  const renderContent = () => {
+    if (location.pathname === Paths.Home) {
+      return <></>;
+    }
+
+    if (isLoading) {
+      return (
+        <>
+          <div>
+            <CurrentLocationWidgetDesktop />
+          </div>
+          <div>
+            <CurrentLocationWidgetMobile />
+          </div>
```

Comment:
**suggestion** If you replace `div` tags to `svg` in styling file, then you can remove those `divs`